### DISCUSSION TITLE
SMite

### DISCUSSION THREAD

**Discussion by ImKruma** [thatskymod/Sky-CotL-Scripts#43] [General]
Smite not working on Sky Beta and Sky Dev Canvas only crashes when I try to activate SMite

**Comment by Sagimoster** [comment]
I put in my sMite script tha canvas was binging 
![Screenshot_2023-10-28-08-49-22-01_2e164e897a11d5b92d17fcbd6532920e](https://github.com/thatskymod/Sky-CotL-Scripts/assets/149201564/119aeb97-092c-4090-9bca-e5c9c85269e8)
![Screenshot_2023-10-28-08-48-55-58_2e164e897a11d5b92d17fcbd6532920e](https://github.com/thatskymod/Sky-CotL-Scripts/assets/149201564/2d54050f-7119-4772-9ec9-25a954039055)
This happen

**Comment by thatskymod** [comment]
These crashes should've been fixed in [libsmite-1.0.7d.so](https://github.com/thatskymod/Sky-CotL-Scripts/blob/main/Canvas/Libs/libsmite-1.0.7d.so) and [Canvas_1.4.3.apk](https://github.com/thatskymod/Sky-CotL-Scripts/blob/main/Canvas/Canvas_1.4.3.apk).

However, if you're still facing any issues, don't hesitate to reach out!
